### PR TITLE
⬆️ Upgrades Grafana to v9.1.5

### PR DESCRIPTION
**Pull request by QuentinBtd**
# Proposed Changes

Upgrades Grafana to v9.1.5

- https://github.com/grafana/grafana/releases/tag/v9.1.5
- https://github.com/grafana/grafana/releases/tag/v9.1.4
- https://github.com/grafana/grafana/releases/tag/v9.1.3
- https://github.com/grafana/grafana/releases/tag/v9.1.2
- https://github.com/grafana/grafana/releases/tag/v9.1.1
- https://github.com/grafana/grafana/releases/tag/v9.1.0

### CODE REVIEW

**Comment by frenck** [comment]
How did you test this?

**Comment by QuentinBtd** [comment]
> How did you test this?

Hi !
You are fast !

I think I am in the wrong way. I am in the process of discovering how to make/update/test addons 😊

Tests will come. Could you add "WIP" label on this PR ?

**Comment by frenck** [comment]
> Tests will come. Could you add "WIP" label on this PR ?

It's your PR. Feel free to mark it as a draft and ready for review again when ready.

File: grafana/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG BUILD_FROM=ghcr.io/hassio-addons/debian-base/amd64:6.0.0
+ARG BUILD_FROM=ghcr.io/hassio-addons/debian-base/amd64:6.1.0
```
Comment (frenck) [inline grafana/Dockerfile:1]: This should go into a separate PR, as it is not directly related to the Grafana upgrade.

**Comment by QuentinBtd** [comment]
It's running on my HAss instance.

(installed via https://github.com/QuentinBtd/hass-repository)

Do you have a check list of specifics tests to make ?

**Comment by fti7** [comment]
Any progress on this PR? :-)

**Comment by QuentinBtd** [comment]
> Any progress on this PR? :-)

Hi,

It run well on my HAss instance since I create this PR. 

I have my dashboard and some plugins installed on it. :)

**Comment by cedricziel** [comment]
@QuentinBtd I think you need to apply a label to have it reviewed again :)

**Comment by cedricziel** [comment]
Maybe update to 9.1.2 while you're at it ;-)

**Comment by QuentinBtd** [comment]
> Maybe update to 9.1.2 while you're at it ;-)

Hi
I'm testing it 👌

**Comment by cedricziel** [comment]
Hi @QuentinBtd - the PR is missing one of the labels and the verify workflow fails.

```
Error! This pull request does not contain any of the valid labels: ['breaking-change', 'bugfix', 'documentation', 'enhancement', 'refactor', 'performance', 'new-feature', 'maintenance', 'ci', 'dependencies', 'translations']
```

**Comment by QuentinBtd** [comment]
> Hi @QuentinBtd - the PR is missing one of the labels and the verify workflow fails.
> 
> ```
> Error! This pull request does not contain any of the valid labels: ['breaking-change', 'bugfix', 'documentation', 'enhancement', 'refactor', 'performance', 'new-feature', 'maintenance', 'ci', 'dependencies', 'translations']
> ```

Hello @cedricziel 
I know that, but I didn’t have the permission to add label 😅

I saw Frenck remove the request review that I asked him. Maybe could you review this PR ?

**Comment by cedricziel** [comment]
I’d love to - however I don’t have permission to.

@frenck we seem to have a locking situation here. Can you guide us how to proceed?

**Comment by frenck** [comment]
Yes, have patience. Thanks 🙏

**Comment by maexrakete** [comment]
Hey!

Curious what the progress is here? I'd really love to use the new Grafana-Version for the better Flux-Query integration :)

Thanks for your work!

**Comment by frenck** [comment]
See my message above. I returned from my holiday this weekend and currently catching up with all work and notifications. As per above, please be patient. Thanks

File: grafana/Dockerfile [hunk 2]
```diff
@@ -29,7 +29,7 @@ RUN \
     && if [ "${BUILD_ARCH}" = "amd64" ]; then \
         apt-get install -y --no-install-recommends \
             libasound2=1.2.4-1.1 \
-            libcups2=2.3.3op2-3+deb11u2 \
+            libcups2=2.3.3op2-3+deb11u2  \
```
Comment (frenck) [inline grafana/Dockerfile:32]: Unrelated change to this PR

```suggestion
            libcups2=2.3.3op2-3+deb11u2 \
```

File: grafana/Dockerfile [hunk 3]
```diff
@@ -8,7 +8,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # Setup base system
 ARG BUILD_ARCH=amd64
 RUN \
-    GRAFANA="8.5.5" \
+    GRAFANA="9.1.2" \
```
Comment (frenck) [inline grafana/Dockerfile:11]:
```suggestion
    GRAFANA="9.1.5" \
```

**Review by frenck — APPROVED** [review]
Thanks, @QuentinBtd 👍
